networking/multiple_networks/secondary_networks/configuring-multi-network-policy.adoc

```diff
@@ -6,8 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Multi-network policy is used by administrators to manage secondary networks created using the network attachment definition (NAD) resource. Administrators use the `MultiNetworkPolicy` API to manage traffic for pods that are attached to multiple networks. For example, you can separate the control plane and data plane traffic.
+Administrators can use the `MultiNetworkPolicy` API to create multiple network policies that manage traffic for pods attached to secondary networks. For example, you can manage traffic between pods attached to the same or different secondary networks by creating policies that allow or deny traffic based on specific ports or labels.
 
+[NOTE]
+====
+Multi-network policies can only manage traffic on secondary networks in the cluster. It cannot manage the cluster's primary network.
```

Comment:
please use either cluster default network and primary-UDNs. I don't know how to interpret "cluster's primary network"...sorry if i am missing something